src/osd/OSD.cc

```diff
@@ -6525,7 +6519,15 @@ void OSD::_dispatch(Message *m)
 
     // -- need OSDMap --
 
-  default:
+  case MSG_OSD_PG_CREATE:
+  case MSG_OSD_PG_NOTIFY:
+  case MSG_OSD_PG_QUERY:
+  case MSG_OSD_PG_LOG:
+  case MSG_OSD_PG_REMOVE:
+  case MSG_OSD_PG_INFO:
+  case MSG_OSD_PG_TRIM:
+  case MSG_OSD_BACKFILL_RESERVE:
+  case MSG_OSD_RECOVERY_RESERVE:
```

Comment:
Huh, does that mean every random message we've received has been grabbing an OSD lock and going through this process?